lua/nvim-tree/git.lua

```diff
@@ -30,8 +32,16 @@ local function update_root_status(root)
   end
 end
 
-function M.get_gitexclude()
-  return vim.fn.system("git ls-files --others --ignored --exclude-standard --directory")
+---Returns a list of all ignored files and directories in the given git directory.
+---@param git_root string|nil
+---@return table
+function M.get_gitignored(git_root)
+  local cmd = "git ls-files --others --ignored --exclude-standard --directory"
+  if git_root then
+    cmd = "cd '" .. git_root .. "' && " .. cmd
```

Comment:
You can pass the cwd as `-C {cwd}` in the git command, if it's empty it defaults to the cwd.

Reply:
Yes, that's better. Good catch.